service/controller/control_plane.go

```diff
@@ -16,7 +18,26 @@ type ControlPlaneConfig struct {
 	K8sClient k8sclient.Interface
 	Logger    micrologger.Logger
 
-	HostAWSConfig aws.Config
+	//TODO LH this should be moved somewhere common
```

Comment:
You want to do this now? Otherwise we tried to put the issues for our TODOs here with the comment so that people can put it in their boards and address it later. Up to you, but either this gets an issue or we solve it.